app/src/main/java/com/jalasoft/routesapp/data/local/room/managers/RouteLocalManager.kt

```diff
@@ -0,0 +1,37 @@
+package com.jalasoft.routesapp.data.local.room.managers
+
+import com.jalasoft.routesapp.data.local.room.db.RoutesDB
+import com.jalasoft.routesapp.data.local.room.interfaces.RouteLocalRepository
+import com.jalasoft.routesapp.data.model.remote.*
+import java.util.*
+
+class RouteLocalManager(private val localRoutesDB: RoutesDB) : RouteLocalRepository {
+
+    override fun getAllLines(): List<LineInfo> {
+        val routePointsStops = localRoutesDB.lineRouteDao().getAllLineRoutePointsStops()
```

Comment:
Same here i think you should get only Lines and Line Routes from the current city selected not all the lines that has been storage

Reply:
Fixed, now the LineInfo list are fetched by the current city id.